### PR TITLE
Handle edge cases

### PR DESCRIPTION
**Pull request by ns1123**
PR to handle edge cases in EMR Bootstrap
- Fix create user if not exists

### CODE REVIEW

File: integration/emr/alluxio-emr.sh [hunk 1]
```diff
@@ -166,12 +166,14 @@ install_alluxio() {
   # Unpack and inflate the release tar
   # TODO logic for different compression formats, s3 URIs, git URIs, etc.
   sudo cp "${release}" /opt/
-  sudo tar -xvf "/opt/${release}" -C /opt/
+  sudo tar -xpvf "/opt/${release}" -C /opt/
```
Comment (ZacBlanco) [inline integration/emr/alluxio-emr.sh]: p?

Comment (ZacBlanco) [reply integration/emr/alluxio-emr.sh]: We chown /opt/alluxio with `alluxio:alluxio` anyways so is this even necessary?

File: integration/emr/alluxio-emr.sh [hunk 2]
```diff
@@ -261,8 +263,7 @@ main() {
   fi
 
   # Create user
-  sudo groupadd alluxio -g 600
-  sudo useradd alluxio -u 600 -g 600
+  id -u alluxio &>/dev/null || sudo useradd alluxio
```
Comment (ZacBlanco) [inline integration/emr/alluxio-emr.sh]: You're gonna need to spell this one out for me

File: integration/emr/alluxio-emr.sh (hunk 1 above)
Comment (ns1123) [reply integration/emr/alluxio-emr.sh]: this is to maintain mode, specifically of the logs/user_log/ directory

File: integration/emr/alluxio-emr.sh (hunk 2 above)
Comment (ns1123) [reply integration/emr/alluxio-emr.sh]: runs both commands. succeeds as long as one of them passes. So either we create the user or the user already exists.

**Comment by ZacBlanco** [comment]
ty x2

**Pull request merged**